terraform/slurm_cluster/modules/_slurm_instance/main.tf

```diff
@@ -20,7 +20,10 @@
 ##########
 
 locals {
-  hostname      = var.hostname == "" ? "default" : var.hostname
```

Comment:
You can assume `hostname` is always provided (please remove defaults from `variables.tf`)
Also `var.hostname_suffix_separator` can be removed, use constant instead.

Reply:
Simplified and `hostname_suffix_separator` removed.